setup1-contributor/src/reliability/cpu.rs

```diff
@@ -0,0 +1,23 @@
+use std::fmt::Debug;
+
+use anyhow::{anyhow, Result};
+use futures::{Sink, SinkExt};
+use setup1_shared::reliability::ContributorMessage;
+use tokio_tungstenite::tungstenite::protocol::Message;
+
+fn calculate_powers_of_tau(input: Vec<u8>) -> Vec<u8> {
```

Comment:
Here the work can be the performance measurement of scalar multiplication operation.

Reply:
Since this PR is just the preview for reliability checks I am approving it. Just wanted to leave this comment as an indication of next steps.